CHANGELOG.md

```diff
@@ -0,0 +1 @@
+## [1.0.0] (IN PROGRESS)
```

Comment:
Please change version to `0.1.0`. New modules require a pre-release step to be approved by technical council, maybe we'll also need to do this for plugin

Reply:
Thanks, fixed